src/client/confirm_history.rs

```diff
@@ -127,6 +129,18 @@ impl ConfirmHistory {
 #[deprecated(note = "use `ConfirmHistory` instead")]
 pub type Confirmed = ConfirmHistory;
 
+/// Triggered for an entity when it receives changes for a tick.
+///
+/// See also [`ConfirmHistory`].
+#[derive(Debug, Event, Clone, Copy)]
+pub struct HistoryConfirmed {
```

Comment:
Maybe rename to `EntityChanged`? The name `HistoryConfirmed` is confusing to me since you haven't necessarily confirmed anything - there could be more mutation messages for this tick or previous ticks.

Reply:
How about `EntityReplicated`? :thinking: `EntityChanged` sounds like something from Bevy.

Reply:
Sure :)

Reply:
How about `MutateTickConfirmed` -> `MutateTickReceived` as well?

Reply:
I like that